spec/features/index_page_spec.rb

```diff
@@ -103,4 +104,33 @@
       expect(page).to have_content('Page data: Past year')
     end
   end
+
+  context 'filter by document_type' do
+    before do
+      content_data_api_has_content_items(from: from, to: to, organisation_id: 'org-id', document_type: 'news_story', items: [items.first])
+      select 'News story', from: 'document_type'
+      click_on 'Filter'
+    end
+
+    it 'makes request to api with correct document_type' do
+      expect(page).to have_content('Content data')
```

Comment:
This test seem to be concerned with implementation detail. Incorrect API calls should be caught by the tests below? Probably can remove this.